package.json

```diff
@@ -248,6 +248,7 @@
     "fix:prettier": "prettier \"**/*.md\" \"**/*.yaml\" \"**/*.[jt]s(x)?\" --write --log-level=error",
     "dev": "(yarn check --verify-tree || yarn install --check-files) && sentry devserver",
     "dev-ui": "SENTRY_UI_DEV_ONLY=1 SENTRY_WEBPACK_PROXY_PORT=7999 webpack serve",
+    "dev-ui-storybook": "SENTRY_UI_DEV_ONLY=1 SENTRY_WEBPACK_PROXY_PORT=7999 STORYBOOK_TYPES=1 webpack serve",
```

Comment:
could we run `dev-ui` with just an additional flag instead of duplicating the task?

```
"dev-ui-storybook": "STORYBOOK_TYPES=1 yarn dev-ui”,
```

Reply:
That's a good call, I don't expect these to diverge much